CHANGES.md

```diff
@@ -80,6 +80,7 @@
 ## Bugfixes
 
 * Fixed incorrect service account impersonation flow for Python pipelines using BigQuery IOs ([#32030](https://github.com/apache/beam/issues/32030)).
+* Updated Go protobuf package to new version (Go) ([#21515](https://github.com/apache/beam/issues/21515)).
```

Comment:
I'd put this under "Improvements" it's not fixing any bugs. It's removing use of deprecated code (that under the hood uses the v2 packages anyway).

Reply:
Done.